proofs/alf/cvc5/programs/Nary.smt3

```diff
@@ -115,23 +142,32 @@
 ; return: >
 ;   The subsequent of t between indices l and u inclusive. Indices that
 ;   fall out of bounds of the number of children of t are ignored.
-(program nary.subsequence
+(program $nary_subsequence
     ((L Type) (cons (-> L L L)) (nil L) (t L) (c1 L) (c2 L :list) (u Int) (l Int))
     ((-> L L L) L Int Int L) Bool
     (
-        ((nary.subsequence cons nil l u nil)            nil)
-        ((nary.subsequence cons nil 0 u t)              (nary.prefix cons nil (alf.add u 1) t))
-        ((nary.subsequence cons nil l u (cons c1 c2))   (nary.subsequence cons nil (alf.add l -1) (alf.add u -1) c2))
+        (($nary_subsequence cons nil l u nil)            nil)
+        (($nary_subsequence cons nil 0 u t)              ($nary_prefix cons nil (alf.add u 1) t))
+        (($nary_subsequence cons nil l u (cons c1 c2))   ($nary_subsequence cons nil (alf.add l -1) (alf.add u -1) c2))
     )
 )
 
-; nary.join
-(program nary.join
+; program: $nary_join
+; args:
+; - cons (-> L L L): The n-ary function symbol.
+; - nil L: The nil terminator of cons.
+; - c1 L: The first term, assumed to be a cons-list.
+; - c2 L: The second term, assumed to be a cons-list.
+; return: >
+;   The "join" of c1 and c2, that is, if the last child is the same as the first
+;   child of c2, then those 2 children are dropped and c1 and c2 are appended
+;   together. Otherwise, this program does not evaluate.
```

Comment:
```suggestion
;   The "join" of c1 and c2, that is, if the last child of c1 is the same as the
;   first child of c2, then those 2 children are dropped and c1 and c2 are
;   appended together. Otherwise, this program does not evaluate.
```